src/corefx/System.Globalization.Native/pal_icushim.c

```diff
@@ -55,7 +55,7 @@ static int FindICULibs(const char* versionPrefix, char* symbolName, char* symbol
 // plus 20 to give us enough headspace. The ICU seems to version about twice
 // a year.
 static const int MinICUVersion = U_ICU_VERSION_MAJOR_NUM;
-static const int MaxICUVersion = MinICUVersion + 20;
+static const int MaxICUVersion = U_ICU_VERSION_MAJOR_NUM + 20;
```

Comment:
What was wrong with the previous way? I'd prefer keeping it as it was.

Reply:
compilation error:

src/corefx/System.Globalization.Native/pal_icushim.c:58:34: error: initializer element is not constant
 static const int MaxICUVersion = MinICUVersion + 20;

Reply:
In C, `MaxICUVersion` is not a constant, it would be a constant in C++, however.

In Globalization C and CoreFX Native, the type-agnostic `#define CONST_NAME CONST_VAL` pattern is used. IMO this could follow the suit:

```c
#define MinICUVersion U_ICU_VERSION_MAJOR_NUM
#define MaxICUVersion MinICUVersion + 20
#define MinMinorICUVersion 1
#define MaxMinorICUVersion 5
#define MinSubICUVersion 1
#define MaxSubICUVersion 5
```

`MinICUVersion + 20` will work as expected: https://repl.it/repls/NeedyMenacingClimate

Reply:
Ah, I keep forgetting about the fact that we have migrated this stuff to C.
@franksinankaya it would be great to migrate to the pattern mentioned by @am11.

Reply:
done